app/celery/process_ses_receipts_tasks.py

```diff
@@ -185,6 +185,22 @@ def process_ses_results(self, response):
                 )
             return
 
+        # Add status reason to notification if the status is some kind of failure
+        if 'temporary-failure' in notification_status:
+            notification.status_reason = "Temporarily failed to deliver email due to soft bounce"
+            notifications_dao.dao_update_notification(notification)
+            current_app.logger.info(
+                "temprary-failure - soft bounce - in process_ses_results for notification %s",
```

Comment:
typo? temprary

Reply:
haha. And this is why I knew I should be copy-pasting whenever I can